source/docs/07-tablature.xml

```diff
@@ -96,50 +89,83 @@
+                  <p>
+                     <figure>
+                        <head/>
+                        <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/tablature/tablature-sample003.txt" parse="text"/></egXML>
+                     </figure>
+                  </p>
+                  <p>Chromatic alteration of the open string’s pitch may be indicated by adding the <att>accid</att>. For example, a guitar in E-flat tuning might look like this:</p>
+                  <p>
+                     <figure>
+                        <head/>
+                        <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/tablature/tablature-sample004.txt" parse="text"/></egXML>
+                     </figure>
+                  </p>
+                  <p>Where the specific instrumental setup is important, the stringing of the instrument can also be specified. The example below is a common way to string and tune a 6-course renaissance lute. The pitch on <gi schema="MEI">course</gi> indicates how the resulting note would normally be transcribed in score, and would often be derived from the lowest sounding pitch.</p>
+                  <p>
+                     <figure>
+                        <head/>
+                        <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/tablature/tablature-sample005.txt" parse="text"/></egXML>
+                     </figure>
+                  </p>
+               </div>
+            </div>
+            <div xml:id="tablature-basics" type="div2">
+               <head>Encoding Tablatures</head>
+               <p>Unlike CMN, <gi schema="MEI">note</gi> elements in tablature do not have an explicit pitch, but indicate playing instructions (i.e. where to place fingers on the fingerboard and when to play the strings). The <att>tab.course</att> attribute is used to capture which course is to be struck, and <att>tab.fret</att> specifies on which fret of the fretboard the player should stop the course. A value of <val>0</val> for <att>tab.fret</att> refers to an unstopped, open course. Course order is the same as that given in the <gi schema="MEI">course</gi> elements.</p>
                <p>
                   <figure>
                      <head/>
-                     <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/tablature/tablature-sample355.txt" parse="text"/></egXML>
+                     <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/tablature/tablature-sample006.txt" parse="text"/></egXML>
```

Comment:
Maybe it could help the reader when some images of the actual tablature could be given next to the encoding in the examples of this subchapter?

Reply:
Agreed -- but we would prefer to save this for MEI v6 :-)